lib/class-convertkit-review-request.php

```diff
@@ -48,14 +57,16 @@ class ConvertKit_Review_Request {
 	 *
 	 * @since   1.9.6.7
 	 *
-	 * @param   string $plugin_name    Plugin Name (e.g. ConvertKit).
-	 * @param   string $plugin_slug    Plugin Slug (e.g. convertkit).
+	 * @param   string      $plugin_name    Plugin Name (e.g. ConvertKit).
+	 * @param   string      $plugin_slug    Plugin Slug (e.g. convertkit).
+	 * @param   bool|string $plugin_path    Plugin Path.
 	 */
-	public function __construct( $plugin_name, $plugin_slug ) {
+	public function __construct( $plugin_name, $plugin_slug, $plugin_path = false ) {
```

Comment:
Why false vs. ""?

Reply:
Should be required, not optional.  Fixed.